examples/correlation_matrices/sampler.cpp

```diff
@@ -147,6 +150,61 @@ void correlation_matrix_uniform_sampling_MT(const unsigned int n, const unsigned
     write_to_file<PointMT>(walkname + "_matrices_MT" + std::to_string(n) + ".txt", randPoints);
 }
 
+template<typename WalkTypePolicy, typename PointType, typename RNGType, typename MT>
+void tune_parameter_L(const int walkL, const int choice, const std::vector<unsigned int>& dimensions, const unsigned int num_points,
+            	      const unsigned int nburns, const unsigned int num_matrices){
+    for (unsigned int n : dimensions){
+        std::list<MT> randCorMatrices;
+   	 
+    	int d = n*(n-1)/2;
+        MT samples(d, num_points);
+        unsigned int jj = 0;
+        for(auto& mat : randCorMatrices){
+            samples.col(jj) = getCoefficientsFromMatrix<NT, MT>(mat);
+            jj++;
+        }
+        double _L;
+    	switch(choice){
+            case 1:  _L = sqrt(d);
+            break;
+            case 2: _L = 2*sqrt(d);
+            break;
+            case 3:  _L=4*sqrt(d);
+            break;
+            case 4:  _L=d/4;
+            break;
+            default: _L=d/10;
+            break;
+	}
+    	std::cout<< "Testing L = " << _L << std::endl;
+    	WalkTypePolicy walk(_L);
+        std::chrono::steady_clock::time_point start, end;
+        double time;
+        start = std::chrono::steady_clock::now();
+
+        uniform_correlation_sampling_MT<WalkTypePolicy, PointType, RNGType>(n, randCorMatrices, walkL, num_points, 0);
```

Comment:
does the `0` corresponds to the nburns?
If yes, we should call `uniform_correlation_sampling_MT<WalkTypePolicy, PointType, RNGType>(n, randCorMatrices, walkL, num_points, nburns);`

Then, the default values for the input variables `nburns` and `walkL` should be 0 and 1 respectively.